src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -727,7 +727,7 @@
   <!-- Instrumentation info attached to BoundBlock -->
   <Node Name="BoundBlockInstrumentation" Base="BoundNode">
     <!-- Local variables with scope that cover the entire block including the instrumentation prologue and epilogue. -->
-    <Field Name="Locals" Type="OneOrMany&lt;LocalSymbol&gt;" Null="disallow"/>
+    <Field Name="Locals" Type="ImmutableArray&lt;LocalSymbol&gt;" Null="disallow"/>
```

Comment:
:memo: The purpose of this change is to get the locals rewritten (see `VisitLocals` in `VisitBlockInstrumentation`). Otherwise, we end up with non-rewritten locals in the emit stage (ie. not found in LocalSlotManager)